File: src/main/java/com/assets/board/model/NbuExchangeRate.java
package com.assets.board.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import lombok.Setter;

@Setter
@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class NbuExchangeRate {

    @JsonProperty("rate")
    String rate;


}

File: src/main/java/com/assets/board/model/NbuExchangeRate.kt
package com.assets.board.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Data
import lombok.Setter

@JsonIgnoreProperties(ignoreUnknown = true)
class NbuExchangeRate {
    @JsonProperty("rate")
    var rate: String? = null
}
